parachain/pallets/inbound-queue/src/benchmarking/mod.rs

```diff
@@ -0,0 +1,70 @@
+mod fixtures;
+
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+use super::*;
+
+use crate::Pallet as InboundQueue;
+use frame_benchmarking::v2::*;
+use frame_system::RawOrigin;
+
+#[benchmarks]
+mod benchmarks {
+	use super::*;
+	use crate::benchmarking::fixtures::make_create_message;
+	use hex_literal::hex;
+
+	const OUTBOUND_QUEUE_ADDRESS: [u8; 20] = hex!["ee9170abfbf9421ad6dd07f6bdec9d89f2b581e0"];
+
+	#[benchmark]
+	fn submit() -> Result<(), BenchmarkError> {
+		let caller: T::AccountId = whitelisted_caller();
+
+		let create_message = make_create_message();
+
+		T::Helper::initialize_storage(
+			create_message.message.proof.block_hash,
+			create_message.execution_header,
+		);
+
+		<AllowList<T>>::put(create_allowlist::<T>());
+
+		let dest_para: ParaId = 1000u32.into();
+		let sovereign_account = dest_para.into_account_truncating();
+
+		// So that the receiving account exists
+		let _ = T::Token::mint_into(&caller, T::Token::minimum_balance().into());
+		// Fund the sovereign account (parachain sovereign account) so it can transfer a reward
+		// fee to the caller account
+		let _ = T::Token::mint_into(&sovereign_account, 10000u32.into());
```

Comment:
Seems like some initialize operation, maybe better move to  where we construct the test runtime [here](https://github.com/Snowfork/snowbridge/blob/9577c65a740479c484062fca4f19eaa835fe74ab/parachain/pallets/inbound-queue/src/test.rs#L167-L175)

Reply:
Vincent mentioned this in yesterday's call, https://github.com/Snowfork/snowbridge/pull/876/files/9577c65a740479c484062fca4f19eaa835fe74ab#diff-d9161fdb08bb2142f833c3f0a33e6592196395f95bb183ba7c372cc432cf741bR52 only gets executed when running the benchmark as a test, not when executing the benchmark. So moving these operations will break the benchmark because the initialize method is only called when executing the tests.